### PR TITLE
Split GraphQL legacy tests and limit usage of Apollo library

### PR DESCRIPTION
**Pull request by olim7t**
`GraphqlApolloTest` is a major pain any time we change something in the CQL-first API. The goal of this PR is to:
* split it into smaller test classes
* make the code more readable
* limit the use of the Apollo client library. Per the discussion on ##270 we don't want to remove it entirely, but if the goal is to validate usage with a "real" client, a couple of requests should be enough.

Work in progress, I've extracted the DDL queries and will continue with the DML ones.

### CODE REVIEW

**Comment by olim7t** [comment]
This ready now.
There are two classes left that I'd still like to convert to the lightweight approach: `BulkInsertTest` and `AtomicDirectiveTest`. But I've spent a lot of time on this PR already, so I'll leave it for a follow-up ticket.

**Review by dougwettlaufer — APPROVED** [review]

File: testing/src/main/java/io/stargate/it/http/graphql/cqlfirst/ErrorsTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@ExtendWith(CqlSessionExtension.class)
+@CqlSessionSpec(initQueries = {"CREATE TABLE \"Foo\"(k int PRIMARY KEY, v int)"})
+public class ErrorsTest extends BaseOsgiIntegrationTest {
```
Comment (tomekl007) [inline testing/src/main/java/io/stargate/it/http/graphql/cqlfirst/ErrorsTest.java:37]: nit: the `ErrorsTest` name seems too generic for me. Can we name it more precisely?

File: testing/src/main/java/io/stargate/it/http/graphql/cqlfirst/BetterbotzTest.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import io.stargate.it.driver.CqlSessionSpec;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/** Base class for GraphQL CQL-first tests that share the data model below. */
+@ExtendWith(CqlSessionExtension.class)
+@CqlSessionSpec(
+    initQueries = {
```
[the remark this answers is not in the document]
Comment (olim7t) [reply testing/src/main/java/io/stargate/it/http/graphql/cqlfirst/BetterbotzTest.java:32]: > we have quite a lot of tables here

There are just 2 tables and 2 insert statements.
I think having it in the code is fine, it avoids the boilerplate to load from a file, and having to hunt down that file when you're reading the code.

File: testing/src/main/java/io/stargate/it/http/graphql/cqlfirst/QueryTest.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.jayway.jsonpath.JsonPath;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class QueryTest extends BetterbotzTestBase {
```
[the remark this answers is not in the document]
Comment (olim7t) [reply testing/src/main/java/io/stargate/it/http/graphql/cqlfirst/QueryTest.java:27]: It is referring to GraphQL queries specifically (as in: fields in the `Query` type), but it's true that the term is overloaded. I'll change it to `SelectTest`.

**Pull request merged**